### DISCUSSION TITLE
Drag and drop between selected contents in the editor

### DISCUSSION THREAD

**Discussion by heqiang123** [eclipse-theia/theia#14358] [Q&A] [answered]
Hello, I have a request here.
Requirement description: The content of the requirement is to implement two editors based on codeEditorWidgets in theia framework. Now we need to drag the selected content in one of the editors to the other editor.
question:
1. Is there any attribute setting in theia that allows content in the editor to be dragged outside the editor?
2. How to monitor when the selected content in the editor is dragged outside the editor?
3. Can theia-monaco set draggable to the selected content tag, and then use native HTML to handle dragging?

**Comment by msujew** [comment, marked as answer]
Hey @heqiang123,

as far as I'm aware the monaco editor (the editor we are using as the underlying editor library) is not capable of dragging text outside the editor. Nevertheless, you should probably ask in the [monaco repository](https://github.com/microsoft/monaco-editor/discussions) directly, since it's more of a monaco question than related to Theia. See also [this existing discussion](https://github.com/microsoft/monaco-editor/discussions/3755).

**Reply by heqiang123** [reply]
thanks for your reply,I've got it
